docs/core/tools/dotnet-install-script.md

```diff
@@ -106,91 +96,45 @@ Specifies the URL for the Azure feed to the installer. It isn't recommended that
 
 If set, the installer uses the proxy when making web requests.
 
-### Bash (macOS/Linux)
-
-`dotnet-install.sh [--channel] [--version] [--install-dir] [--architecture] [--shared-runtime] [--debug-symbols] [--dry-run] [--no-path] [--verbose] [--azure-feed] [--help]`
-
-`-Channel <CHANNEL>`
```

Comment:
The reason why we had the two was to show the options in Windows vs. Unix. In there, it should've been `--channel`.
If we're removing this, should we show the Unix layout above too?

Another option could be to move the descriptions into reusable files so we don't have to have the same text twice and put this content in tabs, similar to https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new?tabs=netcore2x. But instead of .NET Core versions, it would Windows and macOS/Linux. I can help you to do that if you want.

Reply:
The Windows vs. macOS/Linux calling options are shown at lines 23 thru 29 - the parameters have the same meaning in Windows as macOS/Linux

Reply:
If you think that's ok, I'm good with it.